kubernetes-client/src/test/java/io/fabric8/kubernetes/client/internal/UtilsTest.java

```diff
@@ -110,4 +112,36 @@ public void interpolateStringWithParametersTest() {
     // Then
     assertEquals("This is a \"template string\" and the following is code ${NOT_REPLACED}: '1' === '1'; /* END */", result);
   }
+
+  @Test
+  @DisplayName("isNotNull, null, should return false")
+  public void isNotNull() {
+    // When
+    final boolean result1 = Utils.isNotNullOrEmpty((Map)null);
```

Comment:
These are not tests for the implemented method `isNotNull`

You can keep these in a sepearete method `isNotNullOrEmpty` (with changed DisplayName too) to add more coverage, but please do add the nullcheck tests for the currently being implemented method `isNotNull`